python/truth1DxAODAccessor.py

```diff
@@ -71,6 +71,73 @@ def getTruthParticleP4(self, ievent, pdgid, status):
 
         return None
 
+    def getTruthParticleP4Lepton(self, ievent, pdgid, status):
+        """
+        Return the Lorentz vector of the specified truth particle
+        (ROOT.Math.PxPyPzMVector)
+        """
+        truth_particles = self.truth_particles_arr[ievent]
+        full_p4 = []        
+
+        for i, pid in enumerate(truth_particles["TruthParticlesAux.pdgId"]):            
+            if abs(pid) != pdgid:
+                continue
+
+            # check status
+            st = truth_particles["TruthParticlesAux.status"][i]
+            if st != status:
+                continue
+
+            # found the particle
+            # construct Lorentz Vector
+            p4 = ROOT.Math.PxPyPzMVector(
+                truth_particles["TruthParticlesAux.px"][i],
+                truth_particles["TruthParticlesAux.py"][i],
+                truth_particles["TruthParticlesAux.pz"][i],
+                truth_particles["TruthParticlesAux.m"][i]
+                )
+            
+            full_p4.append(p4)
+
+        if len(full_p4) == 0:
+            return None     
+
+        return full_p4    
+
+    def getTruthParticleP4Both(self, ievent, status):
```

Comment:
This function is basically doing the same thing as `getTruthParticleP4Lepton` above, except it has hard-coded 11 and 13 for pdgId. Could we merge this function with `getTruthParticleP4Lepton` to avoid unnecessary duplication? We could make the argument `pdgid` of `getTruthParticleP4Lepton` to take either an int or a list of int.